test/boulder-tools/tag_and_upload.sh

```diff
@@ -23,11 +24,11 @@ do
   echo "Building boulder-tools image ${TAG_NAME}"
 
   # build, tag, and push the image.
-  docker buildx build --build-arg "GO_VERSION=${GO_VERSION}" \
+  time docker buildx build --build-arg "GO_VERSION=${GO_VERSION}" \
```

Comment:
Running `time` for each individual invocation of `docker buildx` probably isn't that useful, since earlier invocations will have the `time` output drowned under the mountains of output from later invocations. Unfortunately there's no nice way from inside a script to say "time this whole script." We could put the whole contents of the script into a big function, but that feels weird. I think it's okay for people to just run `time tag_and_release.sh` when they need to.